CHANGELOG.md

```diff
@@ -2,6 +2,252 @@
 
 Here you can find upgrade changes in between releases and upgrade instructions.
 
+## [1.0]
+
+### [1.0.0] - 2021-05-FIXME
+
+This release includes a security announcement, breaking changes, several new
+features, and more. With the 1.0.0 version of this Helm chart, we look to follow
+[SemVer 2 versioning scheme](https://semver.org/).
+
+#### Highlights
+
+- **arm64 compatible images**
+- **`hub.extraFiles` and `singleuser.extraFiles`**
+- **Automatic secret generation**
+  NOTE about the need to do one upgrade before removing secrets.
+- referencing resources
+  TODO: add documentation section about this
+- (pullOnlyOnChanges)
+- (fullnameOverride)
+
+#### Security announcement
+
+The documentation for how to setup a Amazon EKS cluster included an insecure
+step that would give anyone access to the Kubernetes cluster. If you have
+followed these instructions between `0.7.0-beta.1` and `0.11.1`, please see the
+[this post in the Jupyter forum](TODO).
+
+#### Breaking changes
+
+- **Kubernetes 1.17+ and Helm 3.5+ are now required**
+
+  Helm 3 (3.5+) is now required. Helm 2 reached end of life last year and we
+  have started relying on Helm 3.5 specific features.
+
+  Kubernetes 1.17+ is now required. It helped us avoid maintaining two separate
+  sets of implementations for the the user-scheduler.
+
+- **Schema validation of chart config** ([#2033](https://github.com/jupyterhub/zero-to-jupyterhub-k8s/pull/2033), [#2200](https://github.com/jupyterhub/zero-to-jupyterhub-k8s/pull/2200))
+
+  The Helm chart now bundles with a `values.schema.json` file that will validate
+  all use of the Helm chart during template rendering. If the Helm chart's
+  passed values doesn't comply with the schema, then `helm` will error before
+  the k8s api-server has become involved and anything has changed in the k8s
+  cluster.
+
+  The most common validation errors are:
+
+  - _Unrecognized config values_
+
+    For example if you have misspelled something.
+
+    Note that if you want to pass your custom values for inspection by custom
+    logic in the hub pod, then you should pass these values via the `custom`
+    config section where anything will be accepted.
+
+  - _Recognized config values with the wrong type_
+
+    For example if you have passed a numerical value to a configuration that
+    expected a string.
+
+- default requests:
+  https://github.com/jupyterhub/zero-to-jupyterhub-k8s/pull/2034
+
+  prePuller.resources (0, 0)
+  prePuller.hook.resources (0, 0)
+  scheduling.userScheduler.resources (50m, 256Mi)
+  hub.resources (200m, 512Mi)
+  proxy.resources (200m, 512Mi)
```

Comment:
```suggestion
- **Default resource requests are no longer set** [#2034](https://github.com/jupyterhub/zero-to-jupyterhub-k8s/pull/2034)

  Z2JH now follows Helm chart best practice in not setting default resource limits.
  If you want to restore the old behaviour you can set:
  ```yaml
  hub:
    resources:
      requests:
        cpu: 200m
        memory: 512Mi

  proxy:
    chp:
      resources:
        requests:
          cpu: 200m
          memory: 512Mi

  scheduling:
    userScheduler:
      resources:
        requests:
          cpu: 50m
          memory: 256Mi

  prePuller:
    resources:
      requests:
        cpu: 0
        memory: 0
    hook:
      resources:
        requests:
          cpu: 0
          memory: 0
```
As mentioned on https://github.com/jupyterhub/zero-to-jupyterhub-k8s/issues/2085#issuecomment-846462083 I don't think we need to block this release on having full documentation for resources

Reply:
I updated the suggestion to use four backticks on the suggestion block so that the three backticks within the suggestion wouldn't break the suggestion block.